docs/ReleaseHistory.md

```diff
@@ -218,10 +218,11 @@
 - NEW: Add `Identifiable.ComputeDerivedCommonAnnotatedKey` to generate keys derived from common annotated secrets.
 
 # 1.4.20 - 05/16/2024
+- RUL: Add `SEC101/199.AzureEventGridIdentifiableKey` detection.
```

Comment:
>Add [](http://example.com/codeflow?start=7&length=3)

In fact this was a net new rule for this release, not a conversion of an existing rule like the other two cited below.